enhancements/windows-containers/health-management.md

```diff
@@ -168,6 +176,11 @@ When making use of a PowerShell script, retries should be present within the scr
 is waiting for the HNS network to be created in order to create an HNS endpoint. In that specific case, the PowerShell
 script which creates the endpoint should include a retry period which waits for the needed Network to exist.
 
+For ease of processing, services with the bootstrap flag set to true should have a higher priorities than all 
+non-bootstrap services. 
+Also, when there are multiple services that share the same priority, the tied services' dependencies will be examined to
+help determine order of creation.
```

Comment:
While this is fine, I feel this is a unnecessary complication. You could divide the services into two sections, bootstrap and post-bootstrap (or something named to that effect) and have individual priorities within them.

Reply:
The `tie breaker` logic for multiple services with the same priorities isn't needed. If there's a tie its fine to just go down the list and do them in any arbitrary order. In practice containerd can be priority 0 and kublet priority 1.

Reply:
To be clear, I'd prefer to drop this:
```
the tied services' dependencies will be examined to
help determine order of creation.
```
Instead of making two service lists